README.md

```diff
@@ -1,17 +1,32 @@
-# mcrouter
-[![Build Status](https://travis-ci.org/evertrue/mcrouter-cookbook.svg)](https://travis-ci.org/evertrue/mcrouter-cookbook)
```

Comment:
Reason for removal?
It's quite common for cookbooks to retain build status at top of Readme